README.md

```diff
@@ -187,6 +187,37 @@ result = my_zosmf.get_info()
 ```
 The result will be a JSON object containing z/OSMF information
 
+## Local project setup (Windows)
+Enlisted below are the project setup instructions.
```

Comment:
Thoughts on changing this sentence so that it describes the overall section of this README? E.g.:

"Set up the local project in a Windows environment."